app/assets/javascripts/discourse/models/composer.js.es6

```diff
@@ -709,6 +709,11 @@ const Composer = RestModel.extend({
       const topicProps = this.getProperties(
         Object.keys(_edit_topic_serializer)
       );
+      // frontend should have featuredLink but backend needs featured_link
```

Comment:
this is similar to how we do it for categoryId and category_id. I didn't want to change the variable in front-end to `featured_link`. This also re-uses `_edit_topic_serializer` so that it is easy to add new serialized variables and send them to the backend. I could create topicProps from scratch like ```const topicProps = { title: 'XYZ', .... }``` but it will also mean when adding a new variable in serializer, it needs to be updated here too.